scripts/generate_env_file.py

```diff
@@ -0,0 +1,44 @@
+from io import StringIO
+import os
+import configparser
+import secrets
+
+def read_example_env():
+    file_to_read = None
+    if os.path.isfile('.env'):
+        file_to_read = '.env'
+    else:
+        file_to_read = '.env.example'
+
+    dummy_config = StringIO()
+    dummy_config.write('[dummy]\n')
+    dummy_config.write(open(file_to_read).read())
+    dummy_config.seek(0, os.SEEK_SET)
+
+    cp = configparser.ConfigParser()
+    cp.read_file(dummy_config)
+    return cp['dummy']
+
+def set_if_expected(env, key, expected, value):
+    if env[key] == expected:
+        env[key] = value
+
+def generate_and_set_secrets(env):
+    set_if_expected(env, 'POSTGRES_PASSWORD', 'REPLACE_WITH_SECURE_POSTGRES_PASSWORD', secrets.token_hex(32))
+    set_if_expected(env, 'MEMCACHED_PASSWORD', 'REPLACE_WITH_SECURE_MEMCACHED_PASSWORD', secrets.token_hex(32))
+    set_if_expected(env, 'RABBITMQ_PASSWORD', 'REPLACE_WITH_SECURE_RABBITMQ_PASSWORD', secrets.token_hex(32))
+    set_if_expected(env, 'REDIS_PASSWORD', 'REPLACE_WITH_SECURE_REDIS_PASSWORD', secrets.token_hex(32))
```

Comment:
Do these entropy amounts match what's used in `generate_secrets.py`?  If so, we should have a comment saying so.